<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lab3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="Filter">
          <property name="title">
           <string>Filters</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Cinema</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cinemasBox"/>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>6</width>
                 <height>12</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Session</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="sessionsMinTime"/>
             </item>
             <item>
              <widget class="QTimeEdit" name="sessionsMaxTime"/>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>6</width>
                 <height>12</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Year</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="yearMinBox"/>
             </item>
             <item>
              <widget class="QComboBox" name="yearMaxBox"/>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>6</width>
                 <height>12</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="check2DBox">
               <property name="text">
                <string>2D</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="check3DBox">
               <property name="text">
                <string>3D</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>6</width>
                 <height>12</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Rating</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="ratingMinBox">
               <property name="maximum">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="ratingMaxBox">
               <property name="maximum">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>6</width>
                 <height>12</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Duration</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="durationMinTime">
               <property name="calendarPopup">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="durationMaxTime"/>
             </item>
             <item>
              <spacer name="verticalSpacer_7">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>6</width>
                 <height>12</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Price</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="priceMinBox">
               <property name="maximum">
                <number>99999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="priceMaxBox">
               <property name="maximum">
                <number>99999</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="logsBox">
          <property name="text">
           <string>logs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="importButton">
          <property name="text">
           <string>&amp;Import</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="findField"/>
          </item>
          <item>
           <widget class="QCheckBox" name="fullTextBox">
            <property name="text">
             <string>included</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="findButton">
            <property name="text">
             <string>&amp;Find</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="currentIndex">
           <number>4</number>
          </property>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>&amp;cinemas</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="cinemasView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="showGrid">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>&amp;sessions</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="sessionsView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>&amp;movies</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="moviesView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>&amp;prices</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="pricesView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>&amp;logs</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="logsView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>&amp;Result</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="ResultView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>findField</sender>
   <signal>returnPressed()</signal>
   <receiver>findButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>747</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
